.github/workflows/wheels.yml

```diff
@@ -0,0 +1,74 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This workflow is meant as an extended CI run that includes certain features that shall be tested
+# and JDK versions that are supported but not considered default.
+
+name: "Wheels Test"
```

Comment:
It's better to comment out certain parts of the nightly workflow that are not needed (test runs and packaging/licensing jobs) and run the actual nightly workflow with the wheels creation. That way we actually test the code rather than some other workflow that would be deleted in the end.

Commenting out of parts of the nightly workflow can happen in a separate commit which can be easily dropped before merging the PR.

Reply:
Ahh got it, on my fork the nightly build was all grayed out. I thought this was due to some secret requirement from original repo. But it is a dispatch workflow, I will try with what suggested after addressing other suggestions